templates/base.html

```diff
@@ -21,8 +21,10 @@
 <body>
 <br>
 <div class="container">
+
     <nav class="navbar navbar-expand-lg navbar-light bg-light">
-        <a class="navbar-brand" href="/kiosk">Kiosk</a>
+        <img src="{% static "front_page/shaka.webp" %}" width="70px" alt="shaka">
```

Comment:
For some reason, I can't store this in a global static folder. If I set `STATICFILES_DIRS` Django forces me to remove `STATIC_ROOT`

Reply:
Think this is solved